packages/slo-stat-bot/src/slo-label.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {GitHubAPI} from 'probot/lib/github';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const config = require('./../data/config.json');
+
+interface SLOStatus {
+  appliesTo: boolean;
+  isCompliant: boolean | null;
+}
+
+/**
+ * Function gets ooslo label name in repo from the config file
+ * @returns the name of ooslo label
+ */
+handle_labeling.getLabelName = async function (): Promise<string> {
```

Comment:
Few things on this function:
- Use camelCase for the name
- This function doesn't need to be `async`!  Using `async` comes at the cost of instantiating a promise/closure, so it's not free.  Generally, you only want to use `async` for your function if you need to `await` something inside of it.

Reply:
Thanks for the tip. I will keep that in mind when creating function headers.